packages/providers/news/6069.breaking

```diff
@@ -0,0 +1,3 @@
+Improve and group providers @sneridagh
+Breaking:
+  - The interface of the providers themselves have changed, please check the new one, adapt your apps conviniently
```

Comment:
```suggestion
  - The interface of the providers themselves have changed, please check the new one, adapt your apps accordingly
```

I am just assuming `accordingly` is what you meant but I am not entirely sure it's correct :D ignore this otherwise

Reply:
I corrected a few grammar mistakes, and revised @pnicolli's suggestion further.